package.json

```diff
@@ -58,7 +60,7 @@
         "sass": "^1.56.0",
         "ts-jest": "^29.0.3",
         "ts-node": "^10.9.1",
-        "typescript": "^4.9.3"
-    },
-    "dependencies": {}
+        "typescript": "^4.9.3",
+        "webpack": ">=4"
```

Comment:
Sorry to be a pain, but you know me.

Can we remove the dependency on webpack? This adds a ton of extra stuff and there is an open PR mdx-js to make it optional for usage with nextjs like we are using. 

The other ones are OK.